examples/hosts/app-integration/external-controller/src/app.ts

```diff
@@ -23,16 +22,18 @@ document.title = containerId;
 const dataObjectId = "dice";
 
 async function start(): Promise<void> {
-    const service = new TinyliciousService();
+    const containerConfig: FluidCreateContainerConfig = {
+        id: containerId,
+        dataObjects: [KeyValueDataObject],
+        initialDataObjects: [[dataObjectId, KeyValueDataObject]],
+    };
     // Get or create the document
     const fluidContainer = createNew
-        ? await Fluid.createContainer(service, containerId, [KeyValueDataObject])
-        : await Fluid.getContainer(service, containerId, [KeyValueDataObject]);
+        ? await Fluid.createContainer(containerConfig)
```

Comment:
There's a good case to make here to have `id` outside the config as a property and focus the config on defining what the container will be. Thoughts?

Reply:
I prefer id outside the config so it's reusable.